pyanaconda/ui/tui/spokes/askrd.py

```diff
@@ -114,48 +135,50 @@ def apply(self):
         self.data.vnc.enabled = self._use_rd
 
 
-class VNCPassSpoke(NormalTUISpoke):
+class RDPPassSpoke(NormalTUISpoke):
     """
        .. inheritance-diagram:: VNCPassSpoke
           :parts: 3
     """
 
     def __init__(self, data, storage, payload, message=None):
         super().__init__(data, storage, payload)
-        self.title = N_("VNC Password")
+        self.title = N_("RDP User name & Password")
+        self._username = ""
         self._password = ""
         if message:
             self._message = message
         else:
-            self._message = _("Please provide VNC password (must be six to eight characters long).\n"
-                              "You will have to type it twice. Leave blank for no password")
+            self._message = _("Please provide RDP user name & password.\n"
+                              "You will have to type the password twice.")
 
     @property
     def indirect(self):
         return True
 
     @property
     def completed(self):
-        return True # We're always complete
+        return True  # We're always complete
```

Comment:
Don't you want to check if username is correctly set together with password? What will happen if user will miss `username`?

Reply:
So I've just tried it - it tells you that you have not specified and username & asks again. :)

![no_username](https://github.com/rhinstaller/anaconda/assets/829558/109dcc38-720a-4ebd-84a9-1f409b9fbe40)

So this should be covered. :)